src/pages/Editor.vue

```diff
@@ -106,39 +106,43 @@
               </div>
             </div>
             <div v-else data-test="videoPreview">
-              <div class="relative">
+              <div class="flex relative">
```

Comment:
Why flex?
maximize button will be on top of the video player right? then why do we need to put them in a flexbox?